compiler/rustc_mir_build/src/thir/pattern/deconstruct_pat.rs

```diff
@@ -320,7 +318,8 @@ impl IntRange {
                             ),
                         );
                     }
-                    err.note("this is likely to be a mistake");
+                    err.span_label(pcx.span, "... with this range");
```

Comment:
Ah, this didn't have the effect I hoped it would. I forget how vertical label ordering is determined: I guess by span location?

Reply:
I'm not sure, but I think it's only the macro case that is confusing. In normal cases the ranges will be on different lines of a match and the messages will be in the order we want them

Reply:
Yes, that's true. Not worth worrying about.